nixos/modules/services/misc/devpi-server.nix

```
@@ -0,0 +1,125 @@
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.devpi-server;
+  serverDir = "/var/lib/devpi";
+in
+{
+  options.services.devpi-server = {
+    enable = mkEnableOption (mdDoc "Devpi Server");
```

```suggestion
    enable = mkEnableOption "Devpi Server";
```

mdDoc expresions can be removed